gui/WorldView.cxx

```diff
@@ -810,3 +815,44 @@ void WorldView::updateScale()
     d->renderer->ResetCameraClippingRange();
   }
 }
+
+#ifdef VTKWEBGLEXPORTER
+//-----------------------------------------------------------------------------
+void WorldView::exportWebGLScene(QString const& path)
+{
+  QTE_D();
+
+
+  vtkNew<vtkWebGLExporter> exporter;
+
+  int width = d->renderWindow->GetScreenSize()[0];
+  int height = d->renderWindow->GetScreenSize()[1];
+
+  /** WARNING: This is a workaround. There's a bug in
```

Comment:
> I'll let you know when it's merged and I'll do another PR with the edited comment.

Thanks, but not necessary. Since this was merged, my PR (#178) targets the main repo; @mleotta can merge it directly without needing to go through your repo.